mira/metamodel/ops.py

```diff
@@ -222,9 +221,15 @@ def stratify(
             new_concept = initial.concept.with_context(
                 do_rename=modify_names, **{key: stratum},
             )
-            initials[new_concept.name] = Initial(
-                concept=new_concept, value=initial.value / len(strata),
-            )
+            try:
+                initials[new_concept.name] = Initial(
+                    concept=new_concept, expression=SympyExprStr(initial.expression.args[0] / len(strata))
+                )
+            # handles case where initials.expression.args[0] is of type string vs float
```

Comment:
Under what condition would this happen? I think if initials.expression.args[0] is not sympy.Expr that is a bug somewhere else in the code.